src/main/scala/com/lightbend/conductr/sbt/ConductRPlugin.scala

```diff
@@ -27,7 +33,7 @@ object ConductrPlugin extends AutoPlugin {
   override def trigger = allRequirements
 
   override def globalSettings: Seq[Setting[_]] =
-    super.globalSettings ++ List(
```

Comment:
Settings don't need to declare their super settings as what is declared here will be appended anyhow.